source/stylesheets/mixins/_lists.scss

```diff
@@ -1,6 +1,5 @@
 @mixin plain-list {
-  &,
-  & li {
+  &, & > li {
```

Comment:
I am not sure if it is a general style, just in this case it looked kind of strange to me :wink: